proxied-redirects/www.boundaryproject.io.redirects.js

```diff
@@ -161,7 +161,7 @@ module.exports = [
 	},
 	{
 		source: '/help/admin-ui/worker-filters',
-		destination: '/docs/concepts/filtering/worker-tags',
```

Comment:
Just making sure you don't want to add a new redirect and instead you want to override this legacy one?

Reply:
Yup we want to point to this new redirect since the old one won't be used anymore